ykcompile/src/lib.rs

```diff
@@ -477,7 +477,6 @@ mod tests {
     // A trace which contains a call to something which we don't have SIR for should emit a TIR
     // call operation.
     #[test]
-    #[ignore]
```

Comment:
Does this need to be squashed into the previous commit?

Reply:
That's on purpose. The test belonged in the previous commit, but didn't work because of the mapper bug fixed in this commit.